### PR TITLE
Fix a few typos in Hibernate Search 6.2.0.Alpha2 announcement

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: posts/Yoann/2023-02-14-hibernate-search-6-2-0-Alpha2.adoc [hunk 1]
```diff
@@ -149,7 +149,7 @@ List<Book> hits = searchSession.search( Book.class )
 ----
 [[predicate-bool-new-syntax-root]]
 Simpler `and` predicate for link:{hsearch-doc-url-prefix}#search-dsl-predicate-boolean-lambda[`where( (f, b) -> ... )`] (link:{hsearch-jira-url-prefix}/HSEARCH-4676[HSEARCH-4676])::
-Instead of a `bool` predicate` as in 6.2.0.Alpha1, `where( (f, b) -> ... )` now provides a simpler `and` predicate:
+Instead of a `bool` predicate as in 6.2.0.Alpha1, `where( (f, b) -> ... )` now provides a simpler `add` predicate:
```
Comment (gsmet) [inline posts/Yoann/2023-02-14-hibernate-search-6-2-0-Alpha2.adoc]: Not sure about `and` -> `add` but that's what the example below says.

File: posts/Yoann/2023-02-14-hibernate-search-6-2-0-Alpha2.adoc [hunk 2]
```diff
@@ -174,7 +174,7 @@ List<Book> hits = searchSession.search( Book.class )
 ----
 [[predicate-nested-new-syntax]]
 Simpler `and` predicate for the link:{hsearch-doc-url-prefix}#search-dsl-predicate-nested[`nested` predicate] (link:{hsearch-jira-url-prefix}/HSEARCH-4676[HSEARCH-4676])::
-Instead of a `bool` predicate` as in as in 6.2.0.Alpha1 6.2.0.Alpha1, `f.nested( ... )` now provides a simpler `and` predicate:
+Instead of a `bool` predicate as in as in 6.2.0.Alpha1, `f.nested( ... )` now provides a simpler `add` predicate:
```
Comment (gsmet) [inline posts/Yoann/2023-02-14-hibernate-search-6-2-0-Alpha2.adoc]: Not sure about `and` -> `add` but that's what the example below says.

File: posts/Yoann/2023-02-14-hibernate-search-6-2-0-Alpha2.adoc (hunk 1 above)
Comment (yrodiere) [reply posts/Yoann/2023-02-14-hibernate-search-6-2-0-Alpha2.adoc]: Yeah that's an `and` predicate with `add` methods.

**Comment by yrodiere** [comment]
Merged, thanks

**Pull request merged**